java/build.gradle

```diff
@@ -1,12 +1,17 @@
 // Copyright (c) ZeroC, Inc.
 
+plugins {
+    id 'com.zeroc.ice.slice-tools' apply false
+}
+
 subprojects {
     project.ext.topSrcDir = "$rootProject.projectDir/.."
 
     project.version = iceVersion
     project.group = "com.zeroc"
 
     apply plugin: 'java'
+    apply plugin: 'com.zeroc.ice.slice-tools'
```

Comment:
Should this move into ice.gradle just like the spotless plugin?

Reply:
We can, our gradle files can use a little cleanup.